kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -210,8 +210,11 @@ abstract class KyuubiOperation(session: Session) extends AbstractOperation(sessi
 
   override def setState(newState: OperationState): Unit = {
     MetricsSystem.tracing { ms =>
-      if (!OperationState.isTerminal(state)) {
+      if (!isTerminalState(state)) {
         ms.markMeter(MetricRegistry.name(OPERATION_STATE, opType, state.toString.toLowerCase), -1)
+      } else {
+        val execTime = System.currentTimeMillis() - startTime
```

Comment:
+1 for createTime, it makes sense for customer.

Reply:
I also prefer `createTime`

Reply:
Using `createTime` would duplicate the metric with the `kyuubi.backend_service.xxxx`.
![image](https://github.com/apache/kyuubi/assets/25627922/afd76ddd-c252-4b59-ad74-3b06f4f7ed1b)

Reply:
Then this will introduce another discussion. Since there are `kyuubi.backend_service.xxx` metircs, do we still need to add `exec_time` metric for each Operation?